ports/glutin/window.rs

```diff
@@ -320,7 +319,13 @@ impl Window {
 
     #[cfg(any(target_os = "macos", target_os = "windows"))]
     fn handle_next_event(&self) -> bool {
-        let event = self.window.wait_events().next().unwrap();
+        let event = match self.window.wait_events().next() {
+            None => {
+                warn!("Window event stream closed.");
+                return false;
+            },
```

Comment:
I wasn't sure about what to do here, most likely the window is already closing, and I'm not sure what'll happen if we try to close it twice? Is it idempotent?